src/main/java/com/spotify/ffwd/http/model/v2/ValueDeserializer.java

```diff
@@ -0,0 +1,67 @@
+/*-
+ * -\-\-
+ * FastForward API
+ * --
+ * Copyright (C) 2016 - 2020 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.ffwd.http.model.v2;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.google.protobuf.ByteString;
+import java.io.IOException;
+
+/**
+ * Distribution histogram point value type deserializer
+ * This deserialize supports  {@link Value.DistributionValue}
+ * and {@link Value.DoubleValue} types.
+ */
+public class ValueDeserializer extends StdDeserializer<Value> {
+
+    private static final long serialVersionUID = 6224613339173782914L;
```

Comment:
where does this come from, just curious

Reply:
serialVersionUID is used internally by java for serialization or deserialization. You will get  warning if you don't set this value for classes that implement serializable. .